### ISSUE TITLE
CheckBox and ChoiceFieldGroup: Incorrect state read by JAWS

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-js#44]
_From @mikewheaton on June 5, 2016 22:3_

Here is the bug report:

Navigate to this page: Components/ChoiceField/index.html
Using TAB navigate to the third checkbox
It reads "Checkbox selected, checkboxes unchecked"
It should have read "Checkbox selected, checkbox checked".

Once I pressed spacebar to uncheck and recheck it fixed itself but not on first load.

Same behavior happens for radio buttons

_Copied from original issue: OfficeDev/office-ui-fabric-core#645_

**Comment by Linda-Editor** [comment]
Microsoft no longer supports this content and will not be responding to bugs or issues. We recommend that you use the newer version, [Office UI Fabric](https://developer.microsoft.com/fabric), with React as your front-end framework. We are closing this issue; if you still need assistance with Fabric.js, visit [Stack Overflow/office-ui-fabric](https://stackoverflow.com/questions/tagged/office-ui-fabric).
